alibi/prototypes/tests/test_protoselect.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+import numpy as np
+from sklearn.datasets import make_classification
+from sklearn.model_selection import train_test_split
+
+from alibi.prototypes import ProtoSelect
+from alibi.utils.kernel import EuclideanDistance
+from alibi.prototypes.protoselect import cv_protoselect_euclidean
+
+
+@pytest.mark.parametrize('n_classes', [2, 3, 5, 10])
+@pytest.mark.parametrize('ft_factor', [1, 2, 3, 10])
+@pytest.mark.parametrize('kernel_distance', [EuclideanDistance()])
+@pytest.mark.parametrize('num_prototypes', [30, 40, 50, 100])
+@pytest.mark.parametrize('eps', [0.2, 0.5])
+def test_multimodal(n_classes, ft_factor, kernel_distance, num_prototypes, eps):
+    """ Integration test. """
```

Comment:
Can you clarify why this is called `test_multimodal` ?

Reply:
Supposed to be something else initially and I forgot to rename it.